src/tree_builder/actions.rs

```diff
@@ -775,10 +743,39 @@ impl<Handle, Sink> TreeBuilderActions<Handle>
         // FIXME: application cache selection algorithm
     }
 
+    // https://html.spec.whatwg.org/multipage/syntax.html#create-an-element-for-the-token
```

Comment:
Remove `syntax.html`. However, we really will need comments labelling which steps are being implemented in the code, because it's really hard to figure out the association. I guess the majority of the new code is in support of the form owner stuff in step 12?